tests/lib/uc20-recovery.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash -ex
+
+transition_to_recover_mode(){
+    if [ -n "$1" ]; then
+        local label=$1
+        HAVE_LABEL=1
```

Comment:
we shuould probably have
```
local have_label
```
somewhere earlier

Reply:
done